plugins/BulkTracking/Tracker/Handler.php

```diff
@@ -96,4 +104,23 @@ private function isTransactionSupported()
         return (bool) TrackerConfig::getConfigValue('bulk_requests_use_transaction');
     }
 
+    private function isBulkTrackingRequestAuthenticated(RequestSet $requestSet)
+    {
+        $tokenAuth = $requestSet->getTokenAuth();
+        if (empty($tokenAuth)) {
+            return false;
+        }
+
+        Piwik::postEvent('Request.initAuthenticationObject');
```

Comment:
Sorry just noticed this is only called once, so shouldn't be a big problem I presume